testsuite/MDAnalysisTests/coordinates/test_dcd.py

```diff
@@ -436,3 +437,38 @@ def test_pathlib():
     # we really only care that pathlib
     # object handling worked
     assert u.atoms.n_atoms == 3341
+
+
+@pytest.fixture
```

Comment:
Make a module level fixture so that it really only runs once? Unfortunately will need to use the tmpdir factory